ui/src/dialogs/AboutDialog.js

```diff
@@ -49,15 +49,29 @@ const AboutDialog = ({ open, onClose }) => {
                   <TableCell align="left"> {config.version} </TableCell>
                 ) : (
                   <TableCell align="left">
-                    <Link
-                      href={`https://github.com/navidrome/navidrome/releases/tag/v${
-                        config.version.split(' ')[0]
-                      }`}
-                      target="_blank"
-                      rel="noopener noreferrer"
-                    >
-                      {config.version.split(' ')[0]}
-                    </Link>
+                    {config.version.includes('SNAPSHOT') ? (
+                      <Link
+                        href={`https://github.com/navidrome/navidrome/compare/v${
+                          config.version.split(' ')[0].split('-')[0]
+                        }...${config.version
+                          .split(' ')[1]
+                          .replace(/[()]/g, '')}`}
```

Comment:
The `config.version` should be split into variables, to avoid calling `split` various times, and to handle null references (what if `split['-']` returns only one element?